pkgs/development/python-modules/feedgenerator/default.nix

```
@@ -17,7 +17,7 @@ buildPythonPackage rec {
 
   meta = with stdenv.lib; {
     description = "Standalone version of django.utils.feedgenerator, compatible with Py3k";
-    homepage = https://github.com/dmdm/feedgenerator-py3k.git;
+    homepage = "https://github.com/dmdm/feedgenerator-py3k.git";
```

```suggestion
    homepage = "https://github.com/getpelican/feedgeneratorgit";
```

According to pypi (https://pypi.org/project/feedgenerator/), this should be https://github.com/getpelican/feedgenerator